scripts/us_epa/superfund/sites/tar_creek/README.md

```diff
@@ -0,0 +1,50 @@
+## Contamination measurements at the Tar Creek Superfund site
+[Tar Creek](https://en.wikipedia.org/wiki/Tar_Creek_Superfund_site) is a superfund site located in the Ottawa County, in northeastern Oklahoma, US. The site has groundwater contaminated as a result of mining activities. 
+
+The data on the level of contamination at the site is measured through several sampling wells that are situated within and around the site in the cities of [Picher](https://en.wikipedia.org/wiki/Picher,_Oklahoma) and [Cardin](https://en.wikipedia.org/wiki/Cardin,_Oklahoma). The measurement of contamination levels are published in the 5-year review report mandated by the EPA to monitor the clean up in superfund sites.
+
+#### How to get the dataset:
+For Tar Creek, the 5-year review reports are available as PDF reports at [cumulis.epa.gov](https://cumulis.epa.gov/supercpad/SiteProfiles/index.cfm?fuseaction=second.scs&id=0601269&doc=Y&colid=33990&region=06&type=SC). While the EPA data source is preferred, you can also find these reports at the [Oklahoma Department of Environmental Quality](https://applications.deq.ok.gov/superfundweb/default.aspx?epaid=OKD980629844) page for Tar Creek.
+
+The PDFs are fairly large and requires a pre-processing step to extract the tabular data from the reports, we use [`tabula-py`](https://github.com/chezou/tabula-py) to extract tabular data.
+
+**NOTE:** The extraction of tabular data with `tabula-py` works only for the PDFs downloaded from [cumulis.epa.gov](https://cumulis.epa.gov/supercpad/SiteProfiles/index.cfm?fuseaction=second.scs&id=0601269&doc=Y&colid=33990&region=06&type=SC).
```

Comment:
are there other pdf's from other sources we should be concerned about in this folder? please keep your readme concise..

Reply:
I have added all the PDFs we have for Tar Creek to GitLFS and there are no more PDFs we need.
Noted, I shall remove the repetitions from the README text.